pkgs/tools/system/thinkfan/default.nix

```diff
@@ -25,10 +30,10 @@ stdenv.mkDerivation rec {
     install -Dm644 {src,$out/share/man/man1}/thinkfan.1
   '';
 
-  meta = {
-    license = stdenv.lib.licenses.gpl3;
-    homepage = http://thinkfan.sourceforge.net/;
-    maintainers = with stdenv.lib.maintainers; [ domenkozar ];
-    platforms = stdenv.lib.platforms.linux;
+  meta = with stdenv.lib; {
```

Comment:
please also add a description - left that out earlier, sorry

Reply:
done :-)